build_locally.sh

```diff
@@ -18,12 +27,22 @@ gem install jekyll
 gem install bundler -v 1.17.3
 bundle install
 
-# Build website using similar script as Travis, but skip htmlproofer because it fails locally:
-bundle exec jekyll build --strict_front_matter
+# Test website using same script as Travis
+if have_travis "$@"; then
+  ./.travis.sh
+fi
 
 # Launch website
-echo "Opening http://localhost:4000 in default browser"
-xdg-open http://localhost:4000
```

Comment:
I think you should stick with xdg-open as this opens the default browser of the user.
Some people might not even have google-chrome.

Reply:
Good point, fixed.